programs/library-bridge/Handlers.cpp

```diff
@@ -102,6 +142,8 @@ void LibraryRequestHandler::handleRequest(HTTPServerRequest & request, HTTPServe
             }
 
             const auto & attributes_string = params.get("attributes_names");
+
+            LOG_DEBUG(log, "Parsing attributes names from binary string");
```

Comment:
Quite verbose logging. Maybe we don't need it.

Reply:
I'm already paranoic about this so I'll leave it.